### PR TITLE
Fix typos & small fixes

### PR DESCRIPTION
**Pull request by bretttegartms**

### CODE REVIEW

**Comment by bretttegartms** [comment]
@andreikeis @ceache Ready for review

File: treadmill-scripts/configureldap.sh [hunk 1]
```diff
@@ -116,9 +116,11 @@ trap cleanup EXIT
 cat << E%O%F > ${tmpdir}/LDAP.yaml
 ---
 realm: ${REALM}
+treadmill_cell: None
```
Comment (andreikeis) [inline treadmill-scripts/configureldap.sh]: None is literal string here. you may want to use -, which is translated to python None.

File: treadmill-scripts/configureldap.sh [hunk 2]
```diff
@@ -116,9 +116,11 @@ trap cleanup EXIT
...
 treadmill_dns_domain: ${TREADMILL_DNS_DOMAIN}
 treadmill_ldap: ${TREADMILL_LDAP}
 treadmill_ldap_suffix: ${TREADMILL_LDAP_SUFFIX}
+treadmill_krb_realm: ${REALM}
```
Comment (andreikeis) [inline treadmill-scripts/configureldap.sh]: treadmill_krb_realm or realm?

File: treadmill-scripts/configureldap.sh (hunk 1 above)
Comment (bretttegartms) [reply treadmill-scripts/configureldap.sh]: Updated to use -

File: treadmill-scripts/configureldap.sh (hunk 2 above)
Comment (bretttegartms) [reply treadmill-scripts/configureldap.sh]: Removed duplicate environment variable and updated cloud-init repo to match.

**Pull request merged**